### DISCUSSION TITLE
giving me error when import the package and use it

### DISCUSSION THREAD

**Discussion by rohanthakran** [ritikbanger/react18-input-otp#33] [General]
Invariant Violation: View config getter callback for
 component `input` must be a function (received `undefined`)
. Make sure to start component names with a capital letter.

source 

var agrs =[a,b,c,d,e,f]

**Comment by ritikbanger** [comment]
Please share a minimal reproducable example or your implementation code. It seems like it is the issue with your code and not with the package.

**Comment by ritikbanger** [comment]
References to solve your issue:

https://stackoverflow.com/questions/61523161/view-config-getter-callback-for-component-div-must-be-a-function-received-un

https://heygabriel.medium.com/react-native-view-config-getter-callback-for-component-7449e1ac754b
